src/lib/styles.scss

```diff
@@ -100,3 +100,15 @@
 	margin-inline: auto;
 	padding-inline: var(--pico-spacing);
 }
+
+// Logo
```

Comment:
Just to let you know, the styles for the logo are inside of the Navbar.svelte file on line 95.

Styles.scss is intended for general styles rather than styles specifically for the navbar. Most of the styles in this file are configurations for PicoCSS, the CSS framework we're using for this project.